win32ss/gdi/gdi32/CMakeLists.txt

```diff
@@ -64,6 +64,6 @@ target_link_libraries(gdi32
     atan2
     ${PSEH_LIB})
 
-add_importlibs(gdi32 user32 advapi32 kernel32 ntdll)
+add_importlibs(gdi32 lpk user32 advapi32 kernel32 ntdll)
```

Comment:
Note: For the moment we do this import linking; later we will have to do a similar trick as on Windows, namely a dynamic loading of this dll by gdi32.dll.